00_Base/src/util/request.ts

```diff
@@ -24,6 +24,7 @@ export class RequestBuilder {
     eventGroup: EventGroup,
     origin: MessageOrigin,
     timestamp: Date = new Date(),
+    protocol?: string,
```

Comment:
How about using `OCPPVersion` instead of `string`?

Reply:
Adjusted!